src/postgres/commands/createPostgresDatabase.ts

```diff
@@ -3,13 +3,19 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import * as vscode from 'vscode';
 import { IActionContext } from 'vscode-azureextensionui';
 import { ext } from '../../extensionVariables';
+import { localize } from '../../utils/localize';
+import { PostgresDatabaseTreeItem } from '../tree/PostgresDatabaseTreeItem';
 import { PostgresServerTreeItem } from '../tree/PostgresServerTreeItem';
 
 export async function createPostgresDatabase(context: IActionContext, node?: PostgresServerTreeItem): Promise<void> {
     if (!node) {
         node = <PostgresServerTreeItem>await ext.tree.showTreeItemPicker(PostgresServerTreeItem.contextValue, context);
     }
-    await node.createChild(context);
+    const newDatabase = <PostgresDatabaseTreeItem>await node.createChild(context);
```

Comment:
nit: I'm not sure if the tslint rule requiring types will be turned on soon but it's nice to have types on variables